nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/RunWorker.cs

```diff
@@ -229,7 +229,7 @@ async Task TrackOriginalContentsAsync(string directory, string fileName, string?
                         // TODO: need to report if anything was actually updated
                         if (updateResult.ErrorType is null || updateResult.ErrorType == ErrorType.None)
                         {
-                            if (dependencyLocation != dependencyFilePath)
+                            if (dependencyLocation != dependencyFilePath.EnsurePrefix("/"))
                             {
                                 updatedDependency.Requirements.All(r => r.File == dependencyFilePath);
```

Comment:
Note for future me: what's the purpose of this?  It's computing a `bool` and immediately discarding it.